go.mod

```diff
@@ -1,4 +1,4 @@
-module TUM-Live
+module github.com/joschahenningsen/TUM-Live
```

Comment:
We could possibly remove the requirement for `github.com/joschahenningsen/TUM-Live-Worker-v2 v1.3.3` already.